app/assets/javascripts/angular/controllers/mobile/base-mobile.js

```diff
@@ -0,0 +1,48 @@
+(function() {
+  angular.module('wtsApp').controller('baseMobileCtrl', ['$scope', '$http', function($scope, $http) {
+
+    // Set the default to be no permissions.
+    $scope.myPermissions = [];
+
+    $scope.checkPermissions = function(tab) {
+      var myPermissions = $scope.myPermissions;
+      return myPermissions.indexOf(tab) !== -1;
+    }
+
+    // Tabs
+    $scope.setTab = function(tab) {
+      $scope.focusedTab = tab;
+    }
+
+    var apiCall = function() {
+      $http.get('/api/mobile_basic').then(
+      function successCallback(response) {
+        $('#connection-error').hide();
+
+        var result = response['data']['result'];
+
+        $scope.myCountryId = result['team']['id'];
+        $scope.myCountryName = result['team']['team_name'];
```

Comment:
I'm not sure what's the right behaviour for it to be "fixed" in this case, though - Are teams entered as UK or United Kingdom? 

I suppose I could have the flags in both cases for the moment, but there are so many countries that have multiple names that it would be impractical to store them all - especially as I'd like to move them over to a sprite sheet at some point.